reference/common/models/alerts/base.v1.yaml

```diff
@@ -0,0 +1,17 @@
+# intended as a mix-in to build complete alert specs
+type: object
+properties:
+  enabled:
+    type: boolean
+    default: false
+    description: >-
+      A toggle for disabling the alert.
+  repeat:
```

Comment:
I really like how you designed the "mix-ins" for the alerts. (I'm going to steal this idea for future use. ;) ) Consider making repeat a mixin, too. It just makes it abundantly clear in the alert config that each alert can be repeating.

Definitely not a blocker if you don't want to do.

Reply:
Had "base" contained only repeat, I might have, but it also has the enabled flag. If I make them both mix-ins, then one has to work harder to see what's different about each type of alert. It's a balancing act.

We can always update it later.